packages/neutron-sdk/src/interchain_queries/queries.rs

```diff
@@ -91,8 +77,6 @@ pub fn query_balance(
 ) -> NeutronResult<BalanceResponse> {
     let registered_query = get_registered_query(deps, registered_query_id)?;
 
-    check_query_type(registered_query.registered_query.query_type, QueryType::KV)?;
```

Comment:
The purpose of this function was to check that user don't try to read KV query data from TX query.  You think we don't need this?

Reply:
i think i removed it by mistake, i wanted to remove outer `if let Some(t) = QueryType::try_from_str(&actual)` since `actual` is already `QueryType`. But now i'm thinking, do we need it at all? if user tries to read KV data from TX query, it just fails to reconstruct, right?

Reply:
Yes, but it's an invalid error. I think we should say something "you can't get KV result from a TX query"